common/source/docs/common-loading-firmware-onto-pixhawk.rst

```diff
@@ -51,6 +49,14 @@ Mission Planner will try to detect which board you are using. It may ask you to
 
    Mission Planner: Install FirmwarePrompt
 
+Often you will be presented with a dropdown box of firmware variants for the board, which you can select from (such as bi-directional DShot variants, if available). For boards which share the Pixhawk board id, the list will be extensive, as shown below:
+
+.. image:: ../../../images/pixhawk-firmware.png
+   :target: ../_images/pixhawk-firmware.png
+
+Select the appropriate firmware for your board. For boards marked "Pixhawk", Pixhawk1 firmware is usually the best choice.
+
+.. warning:: Pixhawk1 boards supplied by non ArduPilot partners (ie. "clones") may not be fully compatible with ArduPilot and loading ArduPilot firmware may result in operational issues.
```

Comment:
Maybe we can make this more concrete.  For example, "some Pixhawk boards supplied by non ArduPilot partners (ie. "clones") may have unexpected sensor substitutions which may lead to pre-arm checks or no backup IMU".